com.stansassets.build/Editor/Core/BuildContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEditor;
+
+namespace StansAssets.Build.Editor
+{    
+    /// <summary>
+    /// Build configuration data
+    /// </summary>
+    public class BuildContext : IBuildStepContext
+    {
+        public BuildTarget TargetPlatform => m_TargetPlatform;
+        
+        public BuildPlayerOptions BuildPlayerOptions => m_BuildPlayerOptions;
+
+        public string BuildAlias => TargetPlatform.ToString();
```

Comment:
We going to set BuildAlias directly when starting a build from custom UI or some value by default, I don't see any reason why `public string BuildAlias => TargetPlatform.ToString();` should help us somehow like we have it now.

Reply:
![usestrings](https://user-images.githubusercontent.com/8124139/93191784-986c5380-f74d-11ea-818c-802032727c20.png)
We just use "StandaloneWindows64" or "Android" in public fields of components. And cast it to BuildTarget instead adapters

Reply:
Agree, it can work, but yesterday you pointed a nice case that possible a typo in any setup in a scene and it will break a build (nobody wants that often). So I thought that we can have our "local" enum (Standalone, Mobile) so we can select them in Component Inspector UI (what Michalis will do) and also we need to set alias before build start of course. I guess we can make pretty simple custom editor window and ask everybody make builds from there or just setup different stuff there before the building.